routes/v1.js

```diff
@@ -81,7 +81,7 @@ module.exports = function(appObj) {
 
     return {
         path: '/',
-        api_version: 1,
```

Comment:
Headers tend to be less discoverable, harder to use, and makes it hard to actually change the URL layout (which is the main reason for incrementing the major version in the path).